### PR TITLE
fix #6 galleries size

### LINKED ISSUE #6: possible issue with width of gallery and fixed height for captions
check for in shop with East [hyphen] West item

### PR DESCRIPTION
**Pull request by mixcas**
fix #6

### CODE REVIEW

**Comment by varffs** [comment]
presumably there is a reason it isn't height:auto for desktop? You tested this?

**Comment by mixcas** [comment]
mmmmm not on desktop T_T

ill check

File: css/site.styl [hunk 1]
```diff
@@ -230,6 +230,9 @@ ul
   &:hover
     cursor: pointer
 
+  img
+    height: ((($width-wide-col - ($margin-basic/2)) / 3) * 2)
```
Comment (varffs) [inline css/site.styl]: this causes stretching on my local. `max-height` seems to fix it tho

**Comment by mixcas** [comment]
yeah, i just noticed
i think removeing all height works too, but i cant find the reason why those height where there in first place

**Pull request merged**